blocks/components/editable/index.js

```diff
@@ -1,3 +1,98 @@
-export default function Editable( { value } ) {
-	return wp.element.createElement( 'p', null, value );
+const el = wp.element.createElement;
+let editorInstance = 1;
+
+export default class Editable extends wp.element.Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.onFocusIn = this.onFocusIn.bind( this );
+		this.onFocusOut = this.onFocusOut.bind( this );
+		this.id = `tinymce-instance-${ editorInstance }`;
+		editorInstance++;
+	}
+
+	componentDidMount() {
+		this.initialize();
+		if ( this.props.focusConfig ) {
+			this.focus();
+		}
+	}
+
+	initialize() {
+		const config = {
+			mode: 'exact',
+			elements: this.id,
+			theme: false,
+			inline: true,
+			toolbar: false,
+			entity_encoding: 'raw',
+			setup: this.onSetup,
+			formats: {
+				strikethrough: { inline: 'del' }
+			}
+		};
+
+		tinymce.init( config );
+	}
+
+	onSetup( editor ) {
+		this.editor = editor;
+		editor.on( 'init', this.onInit );
+		editor.on( 'focusIn', this.onFocusIn );
+		editor.on( 'focusout', this.onFocusOut );
+	}
+
+	onInit() {
+		this.editor.setContent( this.props.value );
+	}
+
+	onChange() {
+		const value = this.editor.getContent();
+		if ( value === this.props.value ) {
```

Comment:
- Would [`isDirty`](https://www.tinymce.com/docs/api/tinymce/tinymce.editor/#isdirty) work here?
- Alternatively, would specifying `format: raw` to `getContent` (much more performant) save us at all, or would we expect most calls to this function to pass the condition anyways?